static/app/utils/touch.tsx

```diff
@@ -1,6 +1,4 @@
-function isReactEvent(
-  maybe: MouseEvent | TouchEvent | React.MouseEvent | React.TouchEvent
-): maybe is React.MouseEvent | React.TouchEvent {
+function isReactEvent(maybe: any): maybe is React.SyntheticEvent {
```

Comment:
Losing a bit of type safety here yea?

Reply:
Ah, yes. I think we want `maybe: any` to to be `maybe: MouseEvent | React.SynthethicEvent` here correct? Because nativeEvent will be present in any React.SyntheticEvent